app/controllers/app_controller.rb

```diff
@@ -1,5 +1,21 @@
 class AppController < ApplicationController
   def index
+    @auth_data = check_authenticated
     render 'app'
   end
+
+  private
+
+  def check_authenticated
+    token = cookies.signed[:twitter_session_token]
```

Comment:
This is how i would pass auth data to the frontend, you can do this when frontend and backend are in the same app with rails.